CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# Contributing #
+
+## Conduct ##
+Please refer to [CODE_OF_CONDUCT.md](CODE_OF_CONDUCT.md)
+
+## Process ##
+All contribution is done via forking & pull requests. To get changes merged there must be approval from at least 2 developers at Dubit. Reviews from external developers are welcome and encouraged.
+
+### Reviewing ###
+All reviewers must thoroughly check through any requested changes and comment on points of contention. It's recommended that the changes are checked out, compiled, run and pass any tests before carrying out a more in depth review.
+
+### Communication
+Although we cannot guarantee that changes are reviewed and approved/merged within a particular time frame, we do aim to respond with some communication within 7 days of a new pull request. Communication between contributors and maintainers should be polite & well mannered.
+
+### Releases ###
+We will maintain a 2 week release cycle. Every 2 weeks we will check if any changes have been merged and make a release if warranted. At this point any pending pull requests will be noted and reviews scheduled. These may not be full in-depth reviews, it will depend on the volume of pull-requests and commitments of the Dubit developers.
```

Comment:
We may need to think about this, now with the modularization. 
Maintaining these in a lot of places may be tough, so perhaps we should have a link in here and point all at the core repo's CONTRIBUTING + CODE_OF_CONDUCT